### ISSUE TITLE
Phone gives wrong battey

### ISSUE THREAD

**Issue by PGR-14** [CleverRaven/Cataclysm-DDA#76355]
### Describe the bug

Phones don't give rechargeable light batteries, even though they should

### Attach save file

N/A

### Steps to reproduce

Try to dissemble, get unchargable battery

### Expected behavior

Chargable

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.22631.3880 (23H2)
- Game Version: cdda-experimental-2024-09-10-0532 43dd78a [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    Work in Progress Limb Stuff [limb_wip],
    Translate Complex Dialogue [translate_dialogue],
    Magiclysm [magiclysm],
    Magiclysm Revamp and Additions [magicrevamp],
    Tankmod: Revived [Tankmod_Revived],
    Blazemod - Vehicle Additions Pack [blazemod],
    Extra Mutated Scenarios [extra_mut_scens],
    New start locations for Extra Mutated Scenarios [new_start_locations_for_extra_mutation_scen],
    More Professions can start in Extra Mutated Scenarios [more_profession_chooses_extra_mut_scens],
    More Survivor Notes [more_survivor_notes],
    My Sweet Cataclysm [my_sweet_cataclysm],
    Mind Over Matter [mindovermatter],
    Mind Over Matter: <color_magenta>Remove Rolling Delay in Each Path for Checking Learning Prerequisites</color> [mom_no_power_learn_delay],
    Mind Over Matter: <color_magenta>Remove Pain Disables Powers</color> [mom_remove_pain_disables_psi_limit],
    Mind Over Matter: <color_magenta>Feral Psions Always Drop Crystals</color> [mom_feral_psions_always_crystals],
    Mind Over Matter: <color_magenta>Awakenings Are Always Successful</color> [mom_always_awaken],
    Martial Mastery [perk_melee_system],
    Hands [hand],
    Dream of Flight [dream_of_flight],
    Custom Portal Storm Size [custom_size_ps],
    Custom Frequencey For Portal Storms [custom_freq_ps],
    Bombastic Perks [bombastic_perks],
    Alternative Map Key [alt_map_key],
    Bionic Traits [bionic_traits],
    Targeted Mutagens [targetedmutagens],
    Readd Clothing Customization for TailorKits [TailorKits-Readd-Customization],
    Pixel's Various Fuckery [Pix's Shit],
    Medieval Mod Reborn [Medieval_Mod_Reborn],
    Jackledead Armory [jackledead_armory],
    Frontier Mods Core [frontier_core],
    Gunsmith [gunsmith],
    Culinary Days Ahead [culinary_days_ahead],
    Xedra Evolved [xedra_evolved],
    Smart Cases [hyd_smartcases],
    Make bows great again [bow_opness],
    Jackledead Armory: Creeps Goons Guns [jackledead_armory_expansion],
    Gunslinger Professions [hyd_gunslinger],
    Bionic Professions [package_bionic_professions],
    Better Arachnids [betterArachnids],
    Aftershock [aftershock],
    <color_light_green>Cataclysm++</color> [Cata++],
    Tamable Wildlife [Tamable_Wildlife],
    Stats Through Skills [StatsThroughSkills],
    Stats Through Kills [stats_through_kills],
    SpeedyDex [speedydex],
    Nonperishable Overhaul [Nonperishable_Overhaul],
    No Bionic Slots [no_cbm_slots],
    Autodoc Harmless and Perfect [autodoc_harmless_perfect],
    Additional backgrounds [Arendeth_Additional_backgrounds]
]

**Comment by kevingranade** [comment]
I assume you mean the smart phone since the cell_phone item doesn't yield batteries at all on disassembly.
The smart_phone yields a light_battery_cell, which is not rechargeable, but also I don't think it should even yield that as you're not going to be able to put a smartphone battery pack into another device without some very serious modification.  If anything I'd want it to yield some kind of battery pack that you could use to build a larger battery or possibly incorporate into some crafted electronic device, but will not be directly usable in any consumer devices.

**Comment by PGR-14** [comment]
Fair enough w/ the light battery reasoning. I absolutely agree that we should be able to make a battery pack. Maybe we could make a phone battery &/or laptop battery for crafting those?
